.github/actions/constellation_create/action.yml

```diff
@@ -120,8 +117,7 @@ runs:
         if [[ $output == *"tf-log"* ]]; then
           TFFLAG="--tf-log=DEBUG"
         fi
```

Comment:
I think we can also remove the tf-log flag workaround here, since we dont run tests with CLI versions that didnt support this